frontend/src/components/Forms/ShoutoutsPage/ShoutoutForm.tsx

```diff
@@ -59,29 +54,15 @@ const ShoutoutForm: React.FC<ShoutoutFormProps> = ({ getGivenShoutouts }) => {
   return (
     <Form className={styles.shoutoutForm}>
       <h2 className={styles.formTitle}>Give someone a shoutout! 📣</h2>
-      <label className={styles.formLabel}>
-        Who is awesome? <span className={styles.requiredIcon}>*</span>
-      </label>
-
       <div className={styles.formContainer}>
-        {!recipient ? <MemberSearch onSelect={setRecipient} /> : undefined}
-
-        {recipient ? (
-          <div className={styles.recipientNameDisplayContainer}>
-            <p className={styles.recipientNameDisplay}>
-              {recipient?.firstName} {recipient?.lastName}
-            </p>
-            <Button
-              negative
-              onClick={() => {
-                setRecipient(undefined);
-              }}
-            >
-              Clear
-            </Button>
-          </div>
-        ) : undefined}
-
+        <Form.Input
+          label="Who is awesome?"
+          name="receiver"
+          value={receiver}
+          control={TextArea}
+          onChange={(event) => setReceiver(event.target.value)}
+          required
+        />
```

Comment:
Do you think this could be a simpler input component instead of a text area? It just looks a little strange to have so much space to put a name.
<img width="144" alt="image" src="https://user-images.githubusercontent.com/81320443/197359133-7be345d7-4ef0-4265-b9f9-77fc4a2822b4.png">

Reply:
Yup that makes sense